rtl/eclrtl/eclrtl.cpp

```diff
@@ -3682,6 +3698,13 @@ int rtlSingleUtf8ToCodepage(char * out, unsigned inlen, char const * in, char co
     rtlThrowNoUnicode();
 }
 
+bool rtlCodepageToCodepage(StringBuffer & out, unsigned maxoutlen, unsigned inlen, char const * in, char const * outcodepage, char const * incodepage)
```

Comment:
Shouldn't it throw an exception here?

Reply:
It probably should - I'll check and change

Reply:
On reflection it is probably best to leave as it is - since it will work for strings that only contain ascii characters.
Really it would be better to have some lightweight code that converted from latin-1 to utf8 (it would only require a character mapping table and using the functions in junicode), but that is outside the scope of this fix.